app/src/views/features/sessions/SessionViewer/DraftSessionViewer.tsx

```diff
@@ -62,6 +69,53 @@ const DraftSessionViewer: React.FC = () => {
     ]
   );
 
+  const handleCacheDraftSession = useCallback(
+    async (attributes: RQSessionAttributes, events: RQSessionEvents) => {
+      if (!attributes && !events) return;
+
+      const sessionToCache = {
+        metadata: { name: generateDraftSessionTitle(attributes.url), ...attributes },
```

Comment:
`cacheDraftSession` and `clearDraftSessionCache` can be separated into a separate module?